config-model/src/test/java/com/yahoo/searchdefinition/processing/RankingExpressionTypeValidatorTestCase.java

```diff
@@ -0,0 +1,104 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.searchdefinition.processing;
+
+import com.yahoo.searchdefinition.RankProfileRegistry;
+import com.yahoo.searchdefinition.SearchBuilder;
+import com.yahoo.yolean.Exceptions;
+import org.junit.Test;
+import static com.yahoo.config.model.test.TestUtil.joinLines;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class RankingExpressionTypeValidatorTestCase {
+
+    @Test
+    public void tensorFirstPhaseMustProduceDouble() throws Exception {
+        try {
+            RankProfileRegistry rankProfileRegistry = new RankProfileRegistry();
```

Comment:
Consider creating a helper function that takes the search definition string and builds the SearchBuilder to avoid duplication in the tests.

Reply:
Simplified a bit.